### DISCUSSION TITLE
Share your favorite musical instruments with me!!!

### DISCUSSION THREAD

**Discussion by wangzaixiaoliunai** [NAIST-SE/AttractingContributors#1163] [2024 Attraction Contributions]
Please edit the ReadMe file and send a pull request.
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by ryosuke-sota** [comment]
I made a PR on your repo. </br>Could you contribute to mine? </br>Here : [my project link (fav drinks)](https://github.com/ryosuke-sota/AttractingContributors.git)  </br>Thank you.

**Reply by wangzaixiaoliunai** [reply]
I think I have created a PR

**Comment by 3207-Rhims** [comment]
https://github.com/NAIST-SE/AttractingContributors/pull/1164             Can you write on me also, I think I made the PR

**Reply by wangzaixiaoliunai** [reply]
I have contributed

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI.git
Thank you very much！

**Reply by wangzaixiaoliunai** [reply]
I have contributed

**Reply by anzhitingg** [reply]
Thank you very much！

**Comment by kohanaist** [comment]
I have made a PR on your repo. 
Could you contribute to mine? 
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Reply by wangzaixiaoliunai** [reply]
I have contributed.

**Comment by wangzaixiaoliunai** [comment]
Share your favorite musical instruments with me !!!

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Reply by wangzaixiaoliunai** [reply]
I have contributed
